### DISCUSSION TITLE
听说有的院用脚本打卡被通报了，这个还能用吗？

### DISCUSSION THREAD

**Discussion by Jenix-GH** [lxysl/Auto_Attendance#46] [Q&A]
想知道学校是通过什么来判断的，这个脚本能不能够避免？实在是不想每天都想着打卡这个事。。。

**Comment by lxysl** [comment]
不好意思，我不清楚。本项目仅用于代码学习交流，个人视需求使用，如有外出或感染请如实上报。使用期间产生的任何后果请自行承担，本人恕不负责。
